src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -358,6 +371,10 @@ protected function quoteValue(mixed $value): mixed
             return 'null';
         }
 
+        if ($value instanceof Literal) {
+            return (string)$value;
+        }
```

Comment:
`quoteValue` is used only in the insert functions for dry-run output as well as two queries in the sqlite adapter where it always operates on a string, so not much risk this breaks something.